assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -173,15 +173,13 @@ const renderView = (attributes, toAttribute) => {
 };
 
 export const CoversEditor = ({ attributes, setAttributes, isSelected }) => {
-  const { className, post_types } = attributes;
+  const { className } = attributes;
 
   useEffect(() => {
     const styleClass = getStyleFromClassName(className);
     if (styleClass) {
       setAttributes({
         cover_type: styleClass,
-        posts: [],
```

Comment:
"Content Four Columns" and "Take Action Covers" both use this same attribute, but in the first one it's using posts, and in the second one pages. So that's probably why this was done, to ensure it doesn't get manually picked posts from the wrong type.

With the current code, that would happen again. I think it's less problematic than users losing their input.

Reply:
The only proper solution is if we wouldn't use the block styles for changing what type of data is used. That means re-designing the block editor UX so that we have a separate control for the post type and don't tie this to the block style. It would also mean the block can be used for more cases.